composer.json

```diff
@@ -11,16 +11,22 @@
         }
     ],
     "require": {
-        "silverstripe/framework": "^4.0@dev",
-        "silverstripe/cms": "^4.0@dev"
+        "silverstripe/framework": "~4.0",
+        "silverstripe/cms": "~4.0",
+        "silverstripe/asset-admin": "~1.0.0"
     },
     "require-dev": {
         "phpunit/PHPUnit": "~4.8@stable"
     },
+    "autoload": {
+        "psr-4": {
+            "SilverStripe\\Subsites\\": "code/",
+            "SilverStripe\\Subsites\\Tests\\": "tests/php/"
+        }
+    },
     "extra": {
         "branch-alias": {
             "dev-master": "2.0.x-dev"
         }
-    },
-    "license": "BSD-3-Clause"
```

Comment:
Why was license removed?

Reply:
It's duplicated, towards the top of the file

Reply:
Ah true.